migrations/20220324070441-pluralize-user-place-table.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const [usersPlaces] = await queryInterface.sequelize.query(`select * from "user_place"`, { transaction });
```

Comment:
Just wonder if it would be safer to migrate the data on the DB
```
INSERT INTO users_places (user_id, place_id)
SELECT user_id, place_id FROM user_place;
```